packages/metadata/src/utils.ts

```diff
@@ -0,0 +1,19 @@
+// private function to save text file on local drive
+export function saveTextToFile(textstr: string, filename: string) {
```

Comment:
Is this duplicate code from somewhere else? data module?

Reply:
Yes, this is. My logic for keeping it separate was it seemed clunky to navigate to the directory of the other utils package and I wanted to make the metadata package modular and prevent changes to the utils affecting the metadata package.